src/models/user.js

```diff
@@ -0,0 +1,71 @@
+const mongoose = require('mongoose');
+
+const organizationSchema = new mongoose.Schema(
+  {
+    name: {
+      type: String,
+      required: true,
+    },
+    description: {
+      type: String,
+      required: true,
+    },
+    address: {
+      type: String,
+      required: true,
+    },
+  },
+  { timestamps: true }
+);
+
+const volunteerSchema = new mongoose.Schema(
+  {
+    firstName: {
+      type: String,
+      required: true,
+    },
+    lastName: {
+      type: String,
+      required: true,
+    },
+    birthDate: {
+      type: Date,
+      required: true,
+    },
```

Comment:
Following up from @ibraheemhajjar's suggestion in the previous PR, can you create a virtual field for `age` on this model?